src/App.jsx

```diff
@@ -7,19 +7,30 @@ import CreatePost from "containers/create-post";
 import Home from "containers/home";
 import Login from "containers/login";
 import ProtectedRoute from "containers/protected-route";
-import { addUserToState } from "store/thunks/users";
+import { addUserToState, handleNewUser } from "store/thunks/users";
```

Comment:
`handleNewUser` isn't descriptive enough. Can change to something more readable. is it usersignup? or userlogin?

Reply:
did